scripts/gen_help_html.lua

```diff
@@ -1088,9 +1088,10 @@ local function gen_css(fname)
       padding-bottom: 10px;
       /* Tabs are used for alignment in old docs, so we must match Vim's 8-char expectation. */
       tab-size: 8;
-      white-space: pre;
+      white-space: normal;
       font-size: 16px;
       font-family: ui-monospace,SFMono-Regular,SF Mono,Menlo,Consolas,Liberation Mono,monospace;
+      word-wrap: break-word;
     }
     .old-help-para pre {
       /* All text in .old-help-para is formatted as "white-space:pre" so text following <pre> is
```

Comment:
One minor thing: The comment here doesn't apply here (see #24151), because we no longer have `white-space: pre`, though apparently we'll still need `margin-bottom: 0` in both cases.

Reply:
yes this needs to be fixed.
```suggestion
      /* Text following .old-help-para <pre> is
```

Reply:
I believe that I have updated the pull request with these changes. If I have misinterpreted anything, please let me know.